### PR TITLE
added refresh after artifact form is filled

### PR DESCRIPTION
**Pull request by jamesb2505**

### CODE REVIEW

File: src/components/newartifactform/index.js [hunk 1]
```diff
@@ -26,6 +26,7 @@ function NewArtifactForm({ registerId, showModal, setShowModal }) {
   return (
     <ArtifactForm
       title="Add A New Artifact"
+      F
```
Comment (isubasinghe) [inline src/components/newartifactform/index.js]: Is this meant to be here?

File: src/components/artifactform/index.js [hunk 2]
```diff
@@ -123,6 +125,8 @@ function ArtifactForm({
               date: dateRef.value,
               lat: latRef.value,
               lon: lonRef.value
+            }).then(() => {
+              history.go(0);
```
Comment (isubasinghe) [inline src/components/artifactform/index.js]: Im guessing this is to refetch?

File: src/components/newartifactform/index.js (hunk 1 above)
Comment (jamesb2505) [reply src/components/newartifactform/index.js]: Good catch. Nope

File: src/components/artifactform/index.js (hunk 2 above)
Comment (jamesb2505) [reply src/components/artifactform/index.js]: Yeah. Im not sure if this is okay

Comment (isubasinghe) [reply src/components/artifactform/index.js]: I think it is hacky imo, but we can fix it later

Comment (jamesb2505) [reply src/components/artifactform/index.js]: It is, and it breaks the navbar selection :(

**Comment by isubasinghe** [comment]
Resolve the merge conflict as well btw

**Pull request merged**